Assets/Scripts/MenuButtons.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuButtons : MonoBehaviour {
```

Comment:
new line { (you can change the default unity script template, but it's kind of annoying if you work with lots of different versions and computers)